spatialos-sdk/src/worker/component.rs

```diff
@@ -271,8 +274,20 @@ pub(crate) unsafe fn handle_copy<T>(handle: *mut raw::c_void) -> *mut raw::c_voi
     Arc::into_raw(copy) as *mut _
 }
 
+pub struct WrappedVTable {
+    vtable: Worker_ComponentVtable
+}
+
+impl WrappedVTable {
+    pub fn new(vtable: Worker_ComponentVtable) -> Self {
+        WrappedVTable {
+            vtable
+        }
+    }
+}
+
 // Vtable implementation functions.
-fn create_component_vtable<C: Component>() -> Worker_ComponentVtable {
+pub fn create_component_vtable<C: Component>() -> Worker_ComponentVtable {
```

Comment:
As noted in the other comment, I think it would be better to keep this private and have `VTable::new` call it internally.

Reply:
+1

Reply:
Yeah that makes sense 👍